CMakeLists.txt

```diff
@@ -72,6 +75,12 @@ catkin_package(
         std_msgs
         costmap_2d
         tf2
+        tf2_ros
```

Comment:
There's lot of packages added here - are you sure they're all needed? tf2_ros and tf2_geometry_msgs seems fine - many of these arent in find_package either (conversions, bullet, rostime, etc, I dont think any of these are necessary)